docs/root/intro/arch_overview/load_balancing/overview.rst

```diff
@@ -0,0 +1,54 @@
+.. _arch_overview_load_balancing:
+
+Overview
```

Comment:
Another nice to have would be a glossary here or on the index page.

If you try to walk through top  down, for example, you hit the load balancing types next, which reference priorities, which one reads about after.   Also when I started working on Envoy I had dealt with similar concepts to ours via different naming and it took longer than I'd like to admit to figure out what google-internally-named concept mapped to the envoy-named-concept :-P

Reply:
Yeah I think that would be useful, I can do that in a followup.